ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+Cadre de Décicion Ouverte
+=========================================
+
+Version Communautaire 1.0.3
+
+Dernière mise à jour de la version originale : 13 Oct. 2016
+
+Dernière mise à jour de la traduction : 21 Mars 2017 
+
+© 2014-2016 Red Hat et contributeurs | Le Cadre de Décision Ouverte ("Open
+Decision Framework" en anglais) a été créé par l'équipe Red Hat People et est
+disponible sous licence Creative Commons Attribution-ShareAlike 4.0
+International License ([CC BY-SA
+4.0](https://creativecommons.org/licenses/by-sa/4.0/)). Red Hat et le logo
+Shadowman sont des marques déposées de Red Hat, Inc. et enregistrée dans
+d'autres pays. La marque Red Hat doit être retirée de toute version modifiée.
+
+Notes
+=========
+
+* Cette version markdown des diapositives est publiée pour faciliter la
+  collaboration et le suivi des changements.
+* La numérotation des pages correspond aux fichiers LibreOffice (.odp) et PDF
+  dans le dépot.
+
+Contenu
+===========================================================================
+
+* [Vue d'ensemble](#overview)
+  * [Qu'est-ce qu'une décision ouverte ?](#what-is-an-open-decision)
+  * [Les décisions ouvertes sont issues des principes open source](#open-decisions-open-principles)
+  * [Comment les principes open source conduisent à de meilleures décisions](#how-open-source-principles)
+  * [On ne peut pas plaire à tout le monde](#cant-please-everyone)
+* [Cadre de Décision Ouverte](#framework)
+  * [Phase ou activité: Conceptualiser, Définir, Imaginer](#ideation)
+  * [Phase ou activité: Planifier, Rechercher](#plan-research)
+  * [Phase ou activité: Concevoir, Developper, Tester](#design-develop-test)
+  * [Phase ou activité: Lancer, Deployer, Cloturer](#launch)
+* [Resources](#resources)
+* [Annexes](#appendix)
+  * [Histoire: D'où vient le Cadre de Décision Ouvert](#history)
```

Comment:
It's `Cadre de Décision Ouverte`  the original version states very clearly that the decision itself is open, not just the framework